src/main/java/com/minecolonies/structures/helpers/Settings.java

```diff
@@ -23,13 +23,10 @@
      */
     public              BlockPos                 pos            = null;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Make pos a static final constant or non-public and provide accessors if needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)